### ISSUE TITLE
Update requirements file

### ISSUE THREAD

**Issue by MarvinKweyu** [thu-vu92/python-dashboard-panel#4]
@thu-vu92  I'm curious, why haven't you included `panel` in your requirements file? 

As well, considering a totally new environment does anyone get the error:
```bash
ERROR: The Bokeh notebook application handler requires Jupyter Notebook to be installed.
```

**Comment by stajan81** [comment]
After installing libs from requierements.txt file I had these message:
ERROR: The Bokeh notebook application handler requires Jupyter Notebook to be installed.

I had to install additional libraries to make it work: 
pip3 install jupyter_bokeh
pip3 install nbconvert

**Comment by MarvinKweyu** [comment]
Yeah... I noticed that as well. I'm just curious why they weren't included at the first stance. @thu-vu92  perhaps I can a accept a pull request?

**Comment by Murphy2099** [comment]
Thank for all the code. I was successful to make all the code in Jupyter notebook but when I tried to start the serve. This error appeared:
Error running application handle
 bokeh backend could not plot any Elements in the Overlay
I am using Mac M1

Regards,
